docs/examples/choropleth/example.md

```diff
@@ -92,7 +92,7 @@ css: "#map {
 			fillOpacity: 0.7
 		});
 
-		if (!L.Browser.ie && !L.Browser.opera && !L.Browser.edge) {
+		if (!L.Browser.opera && !L.Browser.edge) {
```

Comment:
BTW, the other two switches here are suspicious. For `edge`, is this detecting the old IE-based Edge, not the Chromium one? And why would Opera not work here if it's also pretty much a flavour of Chrome now? Maybe we could get rid of both of these too.